src/main-process/comfyDesktopApp.ts

```diff
@@ -91,6 +94,20 @@ export class ComfyDesktopApp {
     }
   }
 
+  async checkForUpdates(): Promise<void> {
+    log.info('Checking for updates ...');
+
+    try {
+      const result = await todesktop.autoUpdater?.checkForUpdates();
+      if (result?.updateInfo) {
+        log.info('Update found:', result.updateInfo.version);
```

Comment:
Let's split this function into following 2 functions:
- `checkForUpdates`: Check for update and returns the update info
- `updateToLatest`: Actually restart and do the update